### PR TITLE
[dom] Scalasca 2.3.1 installation for 18.07

### PR DESCRIPTION
**Pull request by kraushm**
NB: Version update of Cube (latest=4.4) and Scalasca (latest=2.4) still have to be done!

### CODE REVIEW

**Comment by kraushm** [comment]
retest this please

File: easybuild/easyconfigs/s/SIONlib/SIONlib-1.7.1-CrayCCE-18.07.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# License::   3-clause BSD
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SIONlib'
+version = '1.7.1'
+# Provide a stripped-down version with renamed symbols for tools,
+# see description for further details
+# versionsuffix = '-tools'
```
Comment (gppezzi) [inline easybuild/easyconfigs/s/SIONlib/SIONlib-1.7.1-CrayCCE-18.07.eb]: remove this?

File: easybuild/easyconfigs/s/Scalasca/Scalasca-2.3.1-CrayPGI-18.07.eb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# Copyright:: Copyright 2013-2016 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+# easyblock = 'EB_Score_minus_P'
```
Comment (gppezzi) [inline easybuild/easyconfigs/s/Scalasca/Scalasca-2.3.1-CrayPGI-18.07.eb]: remove this?

File: easybuild/easyconfigs/s/Scalasca/Scalasca-2.3.1-CrayGNU-18.07.eb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+# Copyright:: Copyright 2013-2016 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+# easyblock = 'EB_Score_minus_P'
```
Comment (gppezzi) [inline easybuild/easyconfigs/s/Scalasca/Scalasca-2.3.1-CrayGNU-18.07.eb]: delete

File: easybuild/easyconfigs/s/Scalasca/Scalasca-2.3.1-CrayIntel-18.07.eb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+# Copyright:: Copyright 2013-2016 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+# easyblock = 'EB_Score_minus_P'
```
Comment (gppezzi) [inline easybuild/easyconfigs/s/Scalasca/Scalasca-2.3.1-CrayIntel-18.07.eb]: delete

**Comment by jgphpc** [comment]
@kraushm thanks for this PR. I will close it and continue in https://github.com/eth-cscs/production/pull/730

**Pull request closed without merging**